vector/src/main/java/im/vector/app/features/location/LocationSharingFragment.kt

```diff
@@ -221,7 +235,9 @@ class LocationSharingFragment @Inject constructor(
     }
 
     private fun startLiveLocationSharing() {
-        viewModel.handle(LocationSharingAction.StartLiveLocationSharing)
+        // TODO. Get duration from user
+        val duration = 30 * 1000L
```

Comment:
might be worth a constant? 

not for this PR - _thinking out of loud about todos_ I'm wondering if they are helpful or add noise, would a `User can set live location duration` ticket that references the class provide the same information :thinking:

Reply:
I already started implementing time-limiting UI. I will create another quick PR today.